core/templates/dev/head/pages/landing/TopicLandingPage.ts

```diff
@@ -16,8 +16,11 @@
  * @fileoverview Controller for landing page.
  */
 
+require(
+  'components/common-layout-directives/common-elements/' +
+  'background-banner.directive.ts');
+
```

Comment:
Please remove this empty line.

Reply:
I was following the guide provided in the wiki page.
Link: https://github.com/oppia/oppia/wiki/Coding-style-guide#adding-require-to-the-typescript-files-with-directive

> The requires for directives that are in HTML are included first and separated from the regular requires by empty line.

Reply:
@vojtechjelinek, we aren't following this rule anymore?  or have I missed something? :p

Reply:
Oh, my bad I didn't realize this is an HTML related require.